buildSrc/src/main/kotlin/Utils.kt

```diff
@@ -0,0 +1,19 @@
+object Projects {
+    const val GROUP = "team.backend"
+    const val VERSIONS = "0.0.1-SNAPSHOT"
+}
+
+object Modules {
+    const val CORE = ":core"
+    const val API = ":api"
+    const val CLIENT = ":client"
+}
+
+object CompilerOptions {
+    const val NULL_SAFETY = "-Xjsr305=strict"
+}
+
+object Resources {
+    const val SOURCE_PATH = "../chaterview-private"
+    const val DESTINATION_PATH = "./src/main/resources"
```

Comment:
I think `RESOURCE_PATH` or `SRC_RESOURCE_PATH` is more appropriate than the name `DESTINATION_PATH`.

Reply:
same above